docs/recipes/real_time_proc.qmd

```diff
@@ -0,0 +1,107 @@
+---
+title: "Real-Time Processing"
+execute:
+  eval: false
+---
+
+
+This recipe serves as an example to showcase the real-time processing capability of DASCore. Here, we demonstrate how to use DASCore to perform rolling mean processing on a spool in real time for edge computing purposes.
+
+
+## Load libraries and get a spool
+
+```{python}
+# Import libraries
+import os
+import time
+
+import dascore as dc
+import numpy as np
+
+from dascore.units import s 
+
+
+# Define path for saving results
+output_data_folder = '/path/to/desired/output/directory'
+
+# Get a spool to work on
+sp = dc.get_example_spool().sort("time").update()
+```
+
+## Set real-time processing parameters (if needed)
+
+In this section, we define the window size and step size required for [rolling](https://dascore.org/api/dascore/proc/rolling/rolling.html) mean processing. With a sampling interval of 10 seconds, the cutoff frequency (Nyquist frequency) is determined to be 0.05 Hz. Additionally, we establish the desired wait time after each run by using the `sleep_time_mult` parameter, which acts as a multiplier coefficient for the number of seconds in each patch.
```

Comment:
The link should use DASCore's internal linking:

```
[rolling](`dascore.proc.rolling.rolling`)
```

That way each doc build will link to its own version of the rolling page rather than pointing to this static URL.